src/hooks/voltz-config/config.ts

```diff
@@ -0,0 +1,271 @@
+import { providers } from 'ethers';
+
+import { NetworkConfiguration } from './types';
+
+export const networkConfigurations: { [key: string]: NetworkConfiguration } = {
```

Comment:
this should be more like `poolConfiguration`

Reply:
yes, I think networkPoolConfigurations because it's still per network.